src/structure_managers/structure_manager_centers.hh

```diff
@@ -326,7 +347,10 @@ namespace rascal {
      */
     std::vector<size_t> offsets{};
 
-    //! Total number of atoms in structure
+    //! Total number of center atoms in the structure
+    size_t n_center_atoms{};
+
+    //! Total number of center atoms in the structure
```

Comment:
A better comment is needed. This is the same comment as for the `n_center_atoms`, but the above means something different, right? This could change based upon the (by me called `mask`).